server/ntb/publish/ntb_nitf.py

```diff
@@ -117,7 +117,7 @@ def _get_filename(self, article):
         example: 2016-08-16_11-07-46_Nyhetstjenesten_Innenriks_ny1-rygge-nedgang.xml
         """
         metadata = {}
-        metadata['date'] = article['versioncreated'].strftime("%Y-%m-%d_%H-%M-%S")
+        metadata['date'] = article['versioncreated'].astimezone().strftime("%Y-%m-%d_%H-%M-%S")
```

Comment:
It's using server's local time zone yes. Actually it was not specified and UTC was used here, but as local timezone is used in other NTB metadata, I've changed it too for consistency.